website/tutorial/basic/12-summary.mdx

```diff
@@ -0,0 +1,11 @@
+---
+id: 11-summary
+title: Summary
+sidebar_label: Summary
+---
+
```

Comment:
Should we mention "Next steps" like:
- Setup GraphQL Code Generator for resolvers
- Caching with dataloader
- Monitoring/Tracing with Envelop
- File uploads with Envelop

Reply:
yes! wanna prep a section with next links? In the future I want to have an advanced tutorial below the basic tutorial.

- Envelop plugins
- Advanced Error modelling (https://graphql.wtf/episodes/30-graphql-error-handling-with-union-types)
- ???